projects/aca-playwright-shared/src/page-objects/pages/favorites.page.ts

```diff
@@ -49,4 +49,10 @@ export class FavoritesPage extends BasePage {
   async waitForPageLoad() {
     await this.page.waitForURL(`**/${FavoritesPage.pageUrl}`);
   }
+
+  async addCommentToNode(commentText: any) {
```

Comment:
seems like this function is unused, also it should have some type for commentText: 
<img width="632" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/4153257b-a2b7-4344-94e6-35153933cb14">

Reply:
sorry, rebase mess-leftovers, this function is declared correctly in another file by @datguychen , removed from here